cinema/serializers.py

```diff
@@ -132,6 +149,30 @@
                 Ticket.objects.create(order=order, **ticket_data)
             return order
 
+    def update(self, instance, validated_data):
+        tickets_data = validated_data.get("tickets", [])
+        instance = super().update(instance, validated_data)
+
+        keep_tickets = []
+        for ticket_data in tickets_data:
+            if "id" in ticket_data.keys():
+                try:
+                    ticket = Ticket.objects.get(id=ticket_data["id"])
+                except Ticket.DoesNotExist:
+                    continue
+                else:
+                    ticket.row = ticket_data.get("row", ticket.row)
+                    ticket.seat = ticket_data.get("seat", ticket.seat)
+                    ticket.save()
+                    keep_tickets.append(ticket.id)
+            else:
+                ticket = Ticket.objects.create(order=instance, **ticket_data)
+                keep_tickets.append(ticket.id)
+
+        instance.tickets.exclude(id__in=keep_tickets).delete()
```

Comment:
The line `instance.tickets.exclude(id__in=keep_tickets).delete()` deletes tickets not in `keep_tickets`. Ensure this behavior is intended, as it will remove any tickets not included in the update request.